frontend/src/routes/Components/QuaternaryButton.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+class QuaternaryButton extends Component {
+  render() {
+    const quaternary={
+      color:"#1A1A1A",
+      borderRadius: "0.625rem",
+      border: "0.0625rem solid #0364FF",
+      bottom:"0rem",
+      right:"22rem"
```

Comment:
what do these do? is this to center the text? or add padding? cuz i think i wouldve done "padding: 1rem" or something and "text-align: center"

Reply:
**This wasn't needed so I removed it.**